docs/notebooks/documents/getting_started.py

```diff
@@ -0,0 +1,159 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.16.1
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # Getting Started
+#
+# This document provides an overview for preparing to work with PDO contracts through the contract
+# docker container. It will walk through the necessary steps to set up a functional
+# configuration. It does assume that the reader is familiar with
+# [basic operation of Jupyter notebooks](https://jupytext.readthedocs.io/en/latest/).
+#
+# %% [markdown]
+# ## How Contracts Use Jupyter
+#
+# Many of the PDO contract families provide Jupyter notebooks to simplify interaction with contract
+# objects. Commonly, the notebooks provide a factory for instantiating a notebook for each new
+# contract from one of the contract specific templates. The templates provided generally include five
+# common sections:
+#
+# * Configure the Contract Object
+# * Initialize the Jupyter Interpreter and the PDO Environment
+# * Initialize the Contract Object Context
+# * Operate on the Contract
+# * View Contract Metadata
+#
+# *Configure the Contract Object*: Each notebook either loads an existing contract object from a
+# file (see the PDO documentation on contract collections) or creates a new contract object based on
```

Comment:
I think "Configure the Contract Object" is perhaps misleading: We are "configuring the notebook" (using the factory notebook as a base) as required to interact with the contract